recipes/libcugraph/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Derived from cugraph build script, as seen here:
+# https://github.com/rapidsai/cugraph/blob/db20b485cfc5399214afcff604b38493f38e83bf/build.sh#L137
+
+CUGRAPH_SRC_DIR=FIXME: set this!
+LIBCUGRAPH_BUILD_DIR=${LIBCUGRAPH_BUILD_DIR:=${CUGRAPH_SRC_DIR}/cpp/build}
+GPU_ARCH=ALL
+INSTALL_PREFIX=${PREFIX:=${CONDA_PREFIX}}
+BUILD_DISABLE_DEPRECATION_WARNING=ON
+BUILD_TYPE=Release
+BUILD_CPP_TESTS=OFF
+BUILD_CPP_MG_TESTS=OFF
+BUILD_STATIC_FAISS=OFF
+PARALLEL_LEVEL=${CPU_COUNT}
+INSTALL_TARGET=install
+VERBOSE_FLAG=""
```

Comment:
We may want to add `export` to all of these

Reply:
Fixed in next commit.